packages/terra-table/src/TableRow.jsx

```diff
@@ -39,14 +61,19 @@ const TableRow = ({
     customProps.className,
   ]);
 
-  const childrenArray = React.Children.toArray(children);
-  if (childrenArray.length > 16) {
-    // eslint-disable-next-line no-console
-    console.log(`Number of Columns are ${React.Children.count(children)}. This is more than columns limit`);
+  const attrSpread = {};
+  if (isSelectable) {
+    attrSpread.onClick = TableUtils.wrappedOnClickForItem(onClick, onSelect, metaData);
+    attrSpread.onKeyDown = TableUtils.wrappedOnKeyDownForItem(onKeyDown, onSelect, metaData);
+    attrSpread.tabIndex = '0';
+    attrSpread['aria-selected'] = isSelected;
```

Comment:
I could be wrong, but do we need to cast isSelected to be a string here for accessibility? 🤔

Reply:
I checked and it looks like the value is converted to the string.